src/handlers/gpt.ts

```diff
@@ -54,6 +55,44 @@ const handleMessageGPT = async (message: Message, prompt: string) => {
 	}
 };
 
+const guessMessage = async (message: Message, prompt: string) => {
```

Comment:
Make guessMessage return an enum (GPT or DALLE)

Then do the logic which one is executed in message.ts (line 90)

I just want to avoid importing handleMessageDALLE in gpt.ts, hope you understand

Reply:
You can make an enum like the following in the types/ folder

```ts
export enum AIType {
    GPT,
    DALLE
}
```

Reply:
👍